apollo-client/src/test/java/com/ctrip/framework/apollo/spi/DefaultConfigFactoryDisableFileCacheTest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2022 Apollo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.ctrip.framework.apollo.spi;
+
+import com.ctrip.framework.apollo.build.MockInjector;
+import com.ctrip.framework.apollo.internals.ConfigRepository;
+import com.ctrip.framework.apollo.internals.RemoteConfigRepository;
+import com.ctrip.framework.apollo.util.ConfigUtil;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+
+public class DefaultConfigFactoryDisableFileCacheTest {
+
+    @Before
+    public void setUp() throws Exception {
+        MockInjector.setInstance(ConfigUtil.class, new MockConfigUtil());
+    }
+
+    @Test
+    public void testCreateFileDisableConfigRepository() throws Exception {
```

Comment:
Add another test to verify the file cache enabled case?

Reply:
added